src/cvode/cvode_proj.c

```diff
@@ -437,15 +438,16 @@ static int cvAccessProjMem(void* cvode_mem, const char* fname, CVodeMem* cv_mem,
   /* Access cvode memory */
   if (cvode_mem == NULL)
   {
-    cvProcessError(NULL, CV_MEM_NULL, "CVODE", fname, MSG_CV_MEM_NULL);
```

Comment:
Same comments as in the ARKODE PR, the `fname` input could be removed and until we have a backtrace the line and func will be a little misleading if the accessor fails.